src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/MathFunction.cs

```diff
@@ -67,9 +105,15 @@ public override bool CheckTypes(CheckTypesContext context, TexlNode[] args, DTyp
             Contracts.Assert(returnType.IsTable);
 
             var arg = args[0];
-            var argType = argTypes[0];
-            fValid &= CheckNumericColumnType(argType, arg, errors, ref nodeToCoercedTypeMap, context, out returnType);
-
+            var argType = argTypes[0];
+
+            if (argType.IsTable)

```

Comment:
>argType.IsTable [](http://example.com/codeflow?start=16&length=15)

looks like previously, it always did the checkcolumntype operation, even if `!argtype.istable`

Reply:
Good call.  The error message was different as a result of the change.  I added an else clause consistent with the two arg version.